client/state/login/actions.js

```diff
@@ -99,6 +109,13 @@ export const loginUser = ( usernameOrEmail, password, redirectTo ) => dispatch =
 					twoStepNonce: get( response, 'body.data.two_step_nonce_sms' ),
 				} );
 			}
+
+			return remoteLoginUser( get( response, 'body.data.token_links', [] ) ).then( () => {
```

Comment:
We should do that **only** if the user doesn't have 2FA, otherwise the user isn't really logged in _yet_